broadcaster/lib/broadcaster/forwarder.ex

```diff
@@ -135,16 +135,25 @@ defmodule Broadcaster.Forwarder do
   end
 
   @impl true
-  def handle_info({:DOWN, _ref, :process, pid, _reason}, %{input_pc: pid} = state) do
+  def handle_info({:DOWN, _ref, :process, pid, reason}, %{input_pc: pid} = state) do
+    Logger.info("Process: #{inspect(pid)} exited with reason: #{inspect(reason)}")
```

Comment:
It would be nice to add information on whether this is the input or output process to the log. The same in the other log.